pkg/osutils/osutils.go

```diff
@@ -123,3 +126,66 @@ func Kubectl(arguments []string) error {
 
 	return nil
 }
+
+// Exec Function to execute local command
+func Exec(command string, arg ...string) (string, error) {
+	out, err := exec.Command(command, arg...).CombinedOutput()
+	outStr := string(out)
+	return outStr, err
+}
```

Comment:
can we use existing RunCommand( ssh.go) or execpod functions

Reply:
This is supposed to be run on the host VM / Jenkins VM where helm / kubectl is already installed and doesn't require to execute on any node. Hence not utilizing the RunCommand which require node object as param.